schema/base/defs.yaml

```diff
@@ -0,0 +1,21 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Base layer properties
+description: Common schema definitions the base theme (primarily from OSM)
+"$defs":
+  propertyDefinitions:
+    wikidata:
+      description: A wikidata ID if available
+      type: string
+      pattern: ^Q\d+
+    sourceTags:
```

Comment:
this is the first property that has type `object` but no fields explicit defined in the schema. i assume parquet type for this would be map<string, string>?

Reply:
Yes